pkg/config/config.go

```diff
@@ -293,6 +294,24 @@ func (c *Config) updateComputedValues() error {
 }
 
 func (c *Config) validate() error {
+	if !isValidIPAddress(c.ApiServer.AdvertiseAddress) {
+		return fmt.Errorf("error validating apiServer.advertiseAddress")
```

Comment:
How about we add info about the value that failed validation?
```suggestion
		return fmt.Errorf("error validating apiServer.advertiseAddress (%q)", c.ApiServer.AdvertiseAddress)
```